integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -213,6 +217,7 @@ public void testDomainLifecyle(Operator operator, Domain domain) throws Exceptio
     domain.verifyDomainCreated();
     domain.verifyWebAppLoadBalancing(TESTWEBAPP);
     domain.verifyAdminServerExternalService(getUsername(), getPassword());
+    domain.verifyHasClusterServiceChannelPort("TCP", 8011, TESTWEBAPP + "/");
```

Comment:
channel port config is there right from the beginning, does it really need domain restart to work?

Reply:
The k8s service for the port/channel is not created until we restart the domain. Even though the port is created in the WLS config from the beginning, the introspector only opens is after the reboot.